solana/solana-ibc/programs/solana-ibc/src/tests.rs

```diff
@@ -3,6 +3,8 @@ use std::str::FromStr;
 use std::thread::sleep;
 use std::time::Duration;
 
+use ::ibc::core::client::types::Height;
+use ::ibc::primitives::Timestamp;
```

Comment:
Both are available through `crtae::ibc` which is already imported. Use `ibc::Height` and `ibc::Timestamp`.

Reply:
sure got it.